app/src/main/java/com/cornellappdev/uplift/ui/components/home/HomeCard.kt

```diff
@@ -0,0 +1,171 @@
+                    Row()
+                    {
+                        Spacer(Modifier.width(12.dp))
+                        Text(
+                            text = gymexample.name,
+                            fontSize = 16.sp,
+                            fontFamily = montserratFamily,
+                            fontWeight = FontWeight(500)
+
+                        )
+                        Spacer(
+                            modifier = Modifier.weight(1F),
+                        )
+                        Image(
+                            painter = painterResource(id = R.drawable.ic_dumbbell),
+                            contentDescription = "Dumbell",
+                        )
+                        Image(
+                            painter = painterResource(id = R.drawable.ic_bowling_pins),
+                            contentDescription = "Bowling Pins",
+                        )
+                        Spacer(modifier = Modifier.width(12.dp))
+                        Spacer(Modifier.height(4.dp))
+                    }
+                    Row() {
+                        Spacer(Modifier.width(12.dp))
+                        if (determineOpen(gymexample.hours)) Text(
+                            text = "Open",
+                            fontSize = 12.sp,
+                            color = Color.Green,
+
+                            fontFamily = montserratFamily,
+                            fontWeight = FontWeight(500),
+                        )
+                        else Text(
+                            text = "Closed",
+                            fontSize = 12.sp,
+                            color = Color.Red,
+
+                            fontFamily = montserratFamily,
+                            fontWeight = FontWeight(500),
+                        )
+                        Spacer(modifier = Modifier.width(8.dp))
+                        /*
+                        @Todo Make this dynamic with networking instead of hardcoding info.
+                         */
+                        Text(
+                            text = "Closes at " + lastTime,
+                            fontSize = 12.sp,
+                            fontFamily = montserratFamily,
+                            fontWeight = FontWeight(500),
+                        )
+
+                    }
+                    Row() {
+                        Spacer(Modifier.width(12.dp))
+                        Text(
+                            text = "Cramped",
+                            fontSize = 12.sp,
+                            fontFamily = montserratFamily,
+                            fontWeight = FontWeight(500),
+                            color = colorResource(id = R.color.orange),
+                            modifier = Modifier.padding(top = 2.dp),
+                        )
+                        Spacer(modifier = Modifier.width(8.dp))
+                        Text(
+                            text = " 120/140",
```

Comment:
Equally tiny nitpick--the color of this text should be `GRAY03` as specified in the Figma. This also follows for the miles text and "closes at" text.

Oh, also the little bowling icons and dumbbell icons should be `GRAY04`. Please change all these over!